### PR TITLE
vl3-nse deployment namespace support

### PR DESCRIPTION
**Pull request by CosminNechifor**
Moved vl3-nse to wcm-system namespace.

Required to be merge with cisco-app-networking/wcm-system#113 changes. 
Closes: cisco-app-networking/cnns-prj#113

### CODE REVIEW

**Comment by tiswanso** [comment]
changes look good.  In the future for PR titles, please use something more specific about the changes.  The title "refactor vl3-nse" made me afraid this was much bigger than the namespace change :-)

**Comment by tiswanso** [comment]
The kiknos NSE needs the same.  Is there an issue for that?

**Review by tiswanso — APPROVED** [review]

File: scripts/vl3/vl3_interdomain.sh [hunk 1]
```diff
@@ -86,20 +87,20 @@ KUBEINSTALL="kubectl $INSTALL_OP ${KCONF:+--kubeconfig $KCONF}"
 CFGMAP="configmap nsm-vl3-${SERVICENAME}"
 if [[ "${INSTALL_OP}" == "delete" ]]; then
     echo "delete configmap"
-    kubectl delete ${KCONF:+--kubeconfig $KCONF} ${CFGMAP}
+    kubectl delete --namespace ${NAMESPACE} ${KCONF:+--kubeconfig $KCONF} ${CFGMAP}
 else
```
Comment (tiswanso) [inline scripts/vl3/vl3_interdomain.sh:91]: Please add a check for existence of the namespace and add the namespace if it doesn't exist.

Comment (CosminNechifor) [reply scripts/vl3/vl3_interdomain.sh:91]: Sure.

**Comment by CosminNechifor** [comment]
> The kiknos NSE needs the same. Is there an issue for that?

From what I know there is no issue for that. We can get those 3 PRs merged and after that we can open another issue and I can pick it up.

**Comment by CosminNechifor** [comment]
> changes look good. In the future for PR titles, please use something more specific about the changes. The title "refactor vl3-nse" made me afraid this was much bigger than the namespace change :-)

Sure thing :+1:

**Review by danielvladco — APPROVED** [review]

File: scripts/vl3/vl3_interdomain.sh [hunk 2]
```diff
@@ -86,20 +87,27 @@ KUBEINSTALL="kubectl $INSTALL_OP ${KCONF:+--kubeconfig $KCONF}"
...
+    wcm_namespace_status=$(kubectl get namespace $NAMESPACE -o=jsonpath='{.status.phase}')
```
Comment (tiswanso) [inline scripts/vl3/vl3_interdomain.sh]: forgot the kubeconfig

File: scripts/vl3/vl3_interdomain.sh [hunk 3]
```diff
@@ -86,20 +87,27 @@ KUBEINSTALL="kubectl $INSTALL_OP ${KCONF:+--kubeconfig $KCONF}"
...
+    if [[ "${wcm_namespace_status}" == "Active" ]]; then
+      echo "Namespace " ${NAMESPACE} " already exists"
+    else
+      kubectl create namespace ${NAMESPACE}
```
Comment (tiswanso) [inline scripts/vl3/vl3_interdomain.sh]: forgot the kubeconfig

**Review by tiswanso — CHANGES_REQUESTED** [review]

File: scripts/vl3/vl3_interdomain.sh (hunk 2 above)
Comment (CosminNechifor) [reply scripts/vl3/vl3_interdomain.sh]: Done. Thanks.

File: scripts/vl3/vl3_interdomain.sh (hunk 3 above)
Comment (CosminNechifor) [reply scripts/vl3/vl3_interdomain.sh]: Thanks.

**Review by danielvladco — APPROVED** [review]

**Review by tiswanso — APPROVED** [review]

**Pull request merged**